### PR TITLE
Set up of input interface and placement of tiles onto board

### PR DESCRIPTION
**Pull request by MichaelKyrollos**
### Done in this PR: 
- generate strings and commands based on user input 
- there are 4 main commands, shown in pictures
- setup of placing tiles onto board 
- methods to display board to user 

### To be completed: - **bolded** = urgent
- **Player class to pass tiles in an arrayList**
- finish commenting code, java doc 
-**logic for placement, is there already a word there? AND if so how to place on board : 
                        - simple 1 liner on board, if square[i][j] = tileonBoard {skip square} 
                        - cannot simply do if  square[i][j] == Null because all squares are not null from the beginning**
- better detection for invalid inputs
- refactor, there are possibilities for reducing lines of code 
- **Add letters and numbers on axis of board**
- **Ask for numbers of players and call function to create that number and start game**

### Showcase
**the words placed do not correspond to the letters inputted yet, implementation is not finished. These letters are randomly generated** 

User controls - help and quit
help

User controls - start (this should print the board, and possibly instructions)
help

Placing vertical word
verticalWord

Placing horizontal word
horizontalWord

### CODE REVIEW

File: src/Board.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+     *
+     * Returns True if word has been placed on board by player, false otherwise.
+     *
+     * @author Michael Kyrollos, ID: 101183521
+     *
+     * @return True if word has been placed, False otherwise.
+     *
+     */
+    public Boolean placeWord(Command command,ArrayList<Tile> tilesToPlay){
+        HashMap<Character,Integer> column = new HashMap<>();
```
Comment (AbsouluteChiller) [inline src/Board.java:38]: Make this a field of Board so it doesn't have to instantiate it every time placeWord() is called.

Comment (AbsouluteChiller) [inline src/Board.java:38]: Line 38-53: Make this a field of Board so it doesn't have to instantiate it every time placeWord() is called.

Comment (AbsouluteChiller) [inline src/Board.java:38]: Line 38-53: Make this a field of Board so it doesn't have to instantiate it every time placeWord() is called.

File: src/Board.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+        char firstChar = command.getThirdWord().charAt(0);
+        char secondChar = command.getThirdWord().charAt(1);
+
+        if (isDigit(firstChar)){
+            int rows = Character.getNumericValue(firstChar);
+            int i =  column.get(secondChar);
+            for (int j = rows; j < tilesToPlay.size() + rows ; j++) {
+                    squares[j][i].placeSquare(tilesToPlay.get(j-rows));}
+            System.out.println(this);
```
Comment (AbsouluteChiller) [inline src/Board.java:63]: Can have a single print outside of the if/else statement

File: src/Square.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+     * TODO Each square can have points
+     *
+     * @author Michael Kyrollos, 101183521
+     * @version 1.0
+     * @date October 22, 2022
+     */
+
+    private Tile tile;
+    public Square() {
+
```
Comment (pathumd) [inline src/Square.java:14]: Consider initializing the only field in the Square class, which is 'tile', to null in the Square constructor.

File: src/Board.java [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+     * scrabble board.
+     *
+     * @author Michael Kyrollos, ID: 101183521
+     *
+     * @return String representing the scrabble game state
+     *
+     */
+    @Override
+    public String toString(){
+        String s = "\n------------------------------------------------------------\n" ;
```
Comment (AbsouluteChiller) [inline src/Board.java:88]: You can start s after the new line with "|", to make the board perfectly symmetrical.

**Review by AbsouluteChiller — APPROVED** [review]

**Review by pathumd — APPROVED** [review]

**Pull request merged**